packages/api/src/utils/psa.js

```diff
@@ -132,11 +149,25 @@ export function transformAndValidate (payload) {
   // Validate CID.
   if (cid) {
     try {
-      normalizedCid = normalizeCid(cid)
-      opts.cid = cid
+      let c
```

Comment:
Nit; I find the following code not easy to read. Not sure why the declaration of c is made here and not in the block below. Could we use only one try? Maybe define a type for the returned struct?